charts/prometheus-statsd-exporter/Chart.yaml

```
@@ -0,0 +1,15 @@
+apiVersion: v2
+name: prometheus-statsd-exporter
+description: A Helm chart for prometheus stats-exporter
+version: 0.1.0
+appVersion: 0.18.0
+home: https://github.com/prometheus/statsd_exporter
+sources:
+  - https://github.com/prometheus/statsd_exporter
```

For Artifact Hub. This link is already used for `home`, let's just add the chart source so people know where to find it. If there are other relevant source links we can add them here too, and label them accordingly.
```suggestion
  - https://github.com/prometheus-community/helm-charts/blob/main/charts/prometheus-statsd-exporter
annotations:
  "artifacthub.io/links": |
    - name: Chart Source
      url: https://github.com/prometheus-community/helm-charts/blob/main/charts/prometheus-statsd-exporter
```